devops/iam-did-auth-proxy-prod/values.yaml

```
@@ -0,0 +1,284 @@
+# Default values for did-auth-proxy-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+did-auth-proxy-helm:
+  enabled: true
+  replicaCount: 1
+
+  image:
+    repository: 984870885661.dkr.ecr.us-west-2.amazonaws.com/did-auth-proxy
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "latest"
+
+  imagePullSecrets: []
+
+  nameOverride: "did-auth-proxy-helm"
+  fullnameOverride: "did-auth-proxy-helm"
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    type: ClusterIP
+    port: 80
+
+  ingress:
+    enabled: false
+
+  config:
+    enabled: false
+    secretRefName: {}
+    configRefName: {}
+
+  appValues:
+    PORT: 80
+    CORS_MAX_AGE: 60
+    CORS_ORIGIN: https://gp4btc-miner.energyweb.org,https://gp4btc-explorer.energyweb.org,http://localhost:4200
```

I think it's better to not allow localhost to connect to prod
```suggestion
    CORS_ORIGIN: https://gp4btc-miner.energyweb.org,https://gp4btc-explorer.energyweb.org
```